ini/windows/bin/sc.ini

```diff
@@ -66,6 +70,10 @@ LINKCMD=%VCINSTALLDIR%\bin\link.exe
 ; C Runtime libraries
 LIB=%LIB%;"%VCINSTALLDIR%\lib\amd64"
 
+; Platform/UCRT libraries (Windows SDK 10.0)
+LIB=%LIB%;"%UniversalCRTSdkDir%\Lib\%UCRTVersion%\um\x64"
+LIB=%LIB%;"%UniversalCRTSdkDir%\Lib\%UCRTVersion%\ucrt\x64"
```

Comment:
What if there is no ucrt dir?
Seems like you should make this conditional using a leading `;UCRT ` or `;VS2015 ` comment.